03-setup-mysql.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Sets up schemas in MySQL database for sample service.
+# Prerequisites:
+# - kubectl is installed and available on the PATH: https://kubernetes.io/docs/tasks/tools/
+# - Kubernetes cluster is configured with kubectl and kubectl context is set to use this cluster
+# - Port forwarding set up to MySQL pod using `kubectl -n mysql port-forward pod/mysql-0 <port>`
+
+tmp_dir=$(mktemp -d)
+trap "rm -rf ${tmp_dir}" EXIT
+
+spire_bundle_file="${tmp_dir}/bundle.pem"
+_goose="goose -certfile ${spire_bundle_file}"
+
+fetch_bundle() {
+    kubectl -n spire exec spire-server-0 -- /opt/spire/bin/spire-server bundle show > "${spire_bundle_file}"
+}
+
+setup_goose_config() {
+    local mysql_pod_name="mysql-0"
+    local mysql_address="127.0.0.1"
+    local mysql_pod_forwarded_port="9999"
```

Comment:
can we use 3306 here so that it maps to the same port of the mysql server?

Reply:
I thought about it but since it's the default port number used by MySQL Server, there's a higher chance of port collision if someone is running MySQL Server locally as well.